test/unit/operations/get_more.test.ts

```diff
@@ -127,11 +93,110 @@ describe('GetMoreOperation', function () {
         operation.execute(server2, session, callback);
       });
     });
+
+    context('command construction', () => {
+      const cursorId = Long.fromBigInt(0xffff_ffffn);
+      const namespace = ns('db.collection');
+      const server = new Server(new Topology([], {} as any), new ServerDescription(''), {} as any);
+
+      it('should build basic getMore command with cursorId and collection', async () => {
+        const getMoreOperation = new GetMoreOperation(namespace, cursorId, server, {});
+        const stub = sinon.stub(server, 'command').callsFake((_, __, ___, cb) => cb());
```

Comment:
I think you can use the more expressive stub API here instead of `.callsFake` either `yields()`, `yieldsRight()` or `callsArg(3)` should all give equivalent results and save the awkwardness of unused underscore placeholders.

Reply:
`calledOnceWith` is magic! much cleaner